### ISSUE TITLE
[Suggestion/Request] sector capture minimum player requirement

### ISSUE THREAD

**Issue by tyl3r99** [KillahPotatoes/KP-Liberation#787]
### Is your feature request related to a problem? Please describe:
 sometimes I've noticed when some groups have their server online majority of the day 1-3 man armies somehow manage to take over sectors. When the main group or a group of players join they find its already nearly completed 

### Describe the solution you'd like:
 It would be very handy feature if we were able to set a minimum player requirement in the sector capture zone to be able to capture the sector thus eliminating the people who do not want to work as a team 

### Additional context:

1. The possibility to be able to disable this feature for those who do not need/want it. 

2. The ability to configure the amount of players required in the sector zone for capture completion (excluding enemy presence)

**Comment by Nopus64** [comment]
good idea, may we can chose between a % value of all players or a fix value of players.
